### PR TITLE
Add enable_mesos_input_plugin config option

### PR DESCRIPTION
**Pull request by gracedo**
https://jira.mesosphere.com/browse/DCOS-47301
Adds the enable_mesos_input_plugin config option.

Terraform Core PR: dcos-terraform/terraform-template-dcos-core#2
DCOS PR: dcos/dcos#4222

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -140,37 +140,33 @@ module "dcos-infrastructure" {
 /////////////////////////////////////////
 
 module "dcos-core" {
-  source  = "dcos-terraform/dcos-install-remote-exec/null"
```
Comment (geekbass) [inline main.tf:143]: set back this source please!

File: main.tf [hunk 2]
```diff
@@ -140,37 +140,33 @@ module "dcos-infrastructure" {
...
-  version = "~> 0.1.0"
```
Comment (geekbass) [inline main.tf]: and this please!

**Review by geekbass — CHANGES_REQUESTED** [review]
LGTM. @fatz 👀

**Comment by gracedo** [comment]
rebased and made the fixes

**Review by bernadinm — APPROVED** [review]
LGTM!
When Jenkins is back up, let's rerun the job before merging if possible.

**Review by geekbass — APPROVED** [review]

**Comment by bernadinm** [comment]
manually ran the test. Here is the output, good to merge:
```
$ terraform fmt
$ terraform validate --check-variables=false
$ docker run -ti -v$(pwd):/a --entrypoint /usr/bin/terraform-docs fatz/terraform-jenkins md /a | tr -d '\r' > README.md
$ git status
On branch gracedo/enable_mesos_input_plugin_config_option
Your branch is up to date with 'origin/gracedo/enable_mesos_input_plugin_config_option'.```

**Pull request merged**